worlds/mlss/__init__.py

```diff
@@ -57,31 +57,26 @@ class MLSSWorld(World):
     location_name_to_id = {loc_data.name: loc_data.id for loc_data in all_locations}
     required_client_version = (0, 5, 0)
 
-    disabled_locations: List[str]
+    disabled_locations: Set[str] = set()
 
     def generate_early(self) -> None:
-        self.disabled_locations = []
         if self.options.skip_minecart:
-            self.disabled_locations += [LocationName.HoohooMountainBaseMinecartCaveDigspot]
+            self.disabled_locations.update([LocationName.HoohooMountainBaseMinecartCaveDigspot])
```

Comment:
I don't really care that much but I thought you might want to know that there is `set.add` for a single item haha